common.xml

```diff
@@ -125,20 +125,38 @@
      choose not to include any version specific code -->
   <property name="version.specific.code.location" location="src/common" />
 
-  <!-- Show pertinent build information -->
-
-  <target name="showIdeaBuild" description="Show build information, including SDK and version data.">
-    <echo>Requested version is set to "${version}"</echo>
-    <echo>Using installed IDEA version of ${idea.sdk.version}</echo>
-    <echo>Using IDEA build at "${idea.ultimate.build}</echo>
-    <echo>Idea property file is ${plugin.properties.file}</echo>
-    <echo>Including version specific code from ${version.specific.code.location}</echo>
+  <!-- Pull apart the build version into component pieces -->
+
+  <script language="javascript">
+    <![CDATA[
```

Comment:
Interesting, I didn't know you could use JavaScript in Ant. Is it with vanilla Ant or you need a plugin ?